src/actions/CommandAction.ts

```diff
@@ -32,18 +35,14 @@ export default class CommandAction implements Action {
       await runOnTargetsForEachEditor<SelectionWithEditor[]>(
         targets,
         async (editor, targets) => {
-          // For command to the work we have to have the correct editor focused
-          if (editor !== window.activeTextEditor) {
-            await focusEditor(editor);
-          }
-
-          const originalSelections = editor.selections;
-
           const targetSelections = targets.map(
             (target) => target.selection.selection
           );
 
-          editor.selections = targetSelections;
+          // For command to the work we have to have the correct editor focused
+          await setSelectionsAndFocusEditor(editor, targetSelections, false);
```

Comment:
When setting selections and focusing editor it is important to first set the selections and then focus the editor. This was fixed in the `setSelectionsAndFocusEditor` function but because it wasn't used here we didn't get that fix